### ISSUE TITLE
Use of Foo-Request-Id as the idempotency header instead of Idempotency-key as used by the Stripe API?

### ISSUE THREAD

**Issue by JamesWismer** [EBSCOIS/api-standards#7]
Stripe is using what seems to be a more appropriately named header for the idempotency key, see:  https://stripe.com/blog/idempotency for reference.

Thanks.

**Comment by esimons** [comment]
Yeah, I would favor a more obvious/explicit header for this. The Square API also calls it an idempotency_key (they require it as part of the POST body, it seems).

**Comment by JamesWismer** [comment]
The Stripe API requires the header for any idempotent post operations (making it part of the API contract).
